rdp-server/rdpgfx.h

```diff
@@ -56,6 +56,7 @@ typedef BOOL (*pfn_rdpgfx_server_cache_to_surface)(rdpgfx_server_context *contex
 
 typedef void (*pfn_rdpgfx_server_open_result)(rdpgfx_server_context* context, rdpgfx_server_open_result result);
 typedef void (*pfn_rdpgfx_server_frame_acknowledge)(rdpgfx_server_context* context, RDPGFX_FRAME_ACKNOWLEDGE_PDU* frame_acknowledge);
+typedef void (*pfn_rdpgfx_server_qoe_frame_acknowledge)(rdpgfx_server_context* context, RDPGFX_QOE_FRAME_ACKNOWLEDGE_PDU* qoe_frame_acknowledge);
```

Comment:
argument can be `const RDPGFX_QOE_FRAME_ACKNOWLEDGE_PDU*`

Reply:
Thanks we'll add a larger const cleanup pr later ...